delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/ResourceShift.scala

```diff
@@ -89,9 +89,10 @@ abstract class ResourceShift[State <: ScopedState, A, M](
   private def toGraphResource(project: ProjectRef, resource: ResourceF[A])(implicit
       cr: RemoteContextResolution
   ): IO[GraphResource] = {
-    val content  = resourceToContent(resource)
-    val metadata = content.metadata
-    val id       = resource.resolvedId
+    implicit val jsonLdOptions = JsonLdOptions.AlwaysEmbed
```

Comment:
Similar question, would this not have potential consequences outside of composite views?